autosklearn/evaluation/__init__.py

```diff
@@ -66,7 +72,7 @@ def fit_predict_try_except_decorator(ta, queue, cost_for_crash, **kwargs):
         queue.close()
 
 
-def get_cost_of_crash(metric):
+def get_cost_of_crash(metric: Scorer) -> int:
```

Comment:
Shouldn't this return float?

Reply:
I got confused with MAXINT, reverted this.